src/picoprobe_config.h

```diff
@@ -62,16 +60,53 @@
 #define PICOPROBE_UART_BAUDRATE 115200
 
 // LED config
-#ifndef PICOPROBE_LED
+#ifdef PICOPROBE_LED
+  #define HAS_FLASHER 1
+#else
+  #if PICO_DEFAULT_LED_PIN == -1
+    #undef PICO_DEFAULT_LED_PIN
+  #endif
+  #ifdef PICO_DEFAULT_LED_PIN
+    #define PICOPROBE_LED PICO_DEFAULT_LED_PIN
+    #define HAS_FLASHER 1
+  #endif
+#endif
 
-#ifndef PICO_DEFAULT_LED_PIN
-#error PICO_DEFAULT_LED_PIN is not defined, run PICOPROBE_LED=<led_pin> cmake
-#elif PICO_DEFAULT_LED_PIN == -1
-#error PICO_DEFAULT_LED_PIN is defined as -1, run PICOPROBE_LED=<led_pin> cmake
+// WS2812 confid
```

Comment:
confid -> config ?

Reply:
Definitely. I was juggling PICOPROBE_LED vs PICOPROBE_DEFAULT_LED at the time.